src/Task.ts

```diff
@@ -85,10 +85,10 @@ export class Task {
     public static readonly dateFormat = 'YYYY-MM-DD';
 
     // Main regex for parsing a line. It matches the following:
-    // - Indentation
+    // - Indentation (including > for potentially nested blockquotes or Obsidian callouts)
```

Comment:
Have you looked for places where `Task.indentation` Is used, in case anything else needs to be updated?

Once place I know it's used is in:
`exclude sub-items`
https://github.com/obsidian-tasks-group/obsidian-tasks/blob/main/src/Query/Filter/ExcludeSubItemsField.ts

I doubt that many people use this instruction, but I think that this change will make every task in a quote or call-out be treated as a sub-item silently.

I'm happy for that to be considered in a separate PR if you wish.